draft-ietf-modpod-group-processes.md

```diff
@@ -84,7 +84,8 @@ after, and subsumes, the moderator team for the IETF discussion list
 {{?RFC9245}} and is tasked to moderate **all** IETF participation channels.
 
 As a consequence, this memo obsoletes {{!RFC3683}} and the "posting rights"
-(PR) action it defines. It also obsoletes {{Section 4 of !RFC9245}}, which
+(PR) action it defines, and {{!RFC3934}}.
+It also obsoletes {{Section 4 of !RFC9245}}, which
```

Comment:
Is there more to do to address #130 or is this it?

Reply:
What we are saying is that guidance must be developed by the moderator team with community input, and then approved, and that admins should act consistently with that guidance.  There is an issue: the guidance won't be there day one.  We might need some interim guidance along the lines of, “Until such time guidance that guidance is approved, admins should use their best judgment."

Reply:
ok, see latest commit.